static/sass/components/_forms.scss

```diff
@@ -0,0 +1,216 @@
+%input_text {
+  @extend .font-sansserif;
+  border: 1px solid $color-gray-light;
+  height: 27px;
+  padding: 0 $size-xxs;
+  color: $color-black-pure;
+  font-size: $size-xs;
+}
+%input_text--l {
+  @extend %input_text;
+  height: 44px;
+  padding: $size-xxs;
+  font-size: $size-b;
+}
+%input_text--placeholder {
+  color: $color-gray-light;
+}
+
+.form {
+  &_instory {
+    border-top: 1px solid $color-gray-light;
+    border-bottom: 1px solid $color-gray-light;
+    margin: $size-b 0;
+    padding: $size-s $size-b $size-b;
+  }
+
+  &_padded {
+    padding: 0 $size-b;
+  }
+
+  ol,
+  ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  li {
+    align-items: center;
+    display: flex;
+    flex-wrap: wrap;
+    margin-bottom: $size-b/2;
+  }
+
+  input[type="checkbox"],
+  input[type="radio"] {
+    flex: none;
+    margin-right: $size-b/2;
+  }
+
+  input[type="email"]::placeholder {
+    font-size: $size-xs;
+  }
+
+  input[type="email"]:focus,
+  input[type="email"]:valid {
+    background-color: $color-white-pure;
+  }
+
+  input[type="text"] {
+    @extend %input_text;
+  }
+
+  fieldset {
+    border: 0;
+  }
+
+  &_labels {
+    padding-top: $size-b;
+
+    li {
```

Comment:
This is my fault (because I originally wrote this) but it's kinda smelly to put so many styles on the base elements inside a form, even with a class... I did this because there were so many form templates to rewrite & didn't want to bother adding classes to all the templates. Not super important, but we could work on refactoring that here since its in a smaller context and later pull those updated styles/templates into the main repo.

Reply:
@brickaa Totally agree. I think we'll be able to address this after Andrew's refactor because all of the HTML won't be buried in the wtf forms. Okay if we hold off on this fix until after that?

cc @AndrewGibson27